hedera-node/test-clients/src/main/resources/contract/contracts/GetScheduleInfo/HederaScheduleService.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Does this file need to be duplicated here (from `../HederaScheduleService`?  Can't it be imported with a `..` over at `GetSchedule.sol`?  Or is this something to do with the way the hapi spec framework works?  (And if so, can we fix it?). 

(Same for `IHederaScheduleService.sol` too.)

Reply:
My guess it is necessary to compile.

Reply:
You can't compile it anyway:`GetScheduleInfo.sol` also has `import "./HederaResponseCodes.sol"` and that isn't here in the same directory, and it also uses `IHederaTokenService.sol` (indirectly) which isn't here.

The instructions at `how-to-compile-contracts.txt` are wrong and needs to have a sufficient set of `--include-path` arguments.  Including, at least, `IHederaTokenService`, and someplace for the `HederaResponseCodes.sol`.  I'll file an issue.